src/GitVersion.Core.Tests/Helpers/ConfigBuilder.cs

```diff
@@ -0,0 +1,161 @@
+using GitVersion.Configuration;
+using GitVersion.Model.Configuration;
+using GitVersion.VersionCalculation;
+
+namespace GitVersion.Core.Tests.Helpers;
+
+public sealed class ConfigBuilder
```

Comment:
just rename this one to TestConfigurationBuilder

Reply:
Done